include/functions.h

```diff
@@ -252,4 +256,10 @@ void func_80070B50(s32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4);
 void func_800718D0(s32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4, s32 arg5);
 void func_80070BB0(s32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4, s32 arg5);
 
+// Dead functions:
```

Comment:
these should probably definitely get their own header that the dead sections explicitly include

Reply:
maybe prefix em with dead_ too? like `dead_atan2`